lib/Models/GlobeOrMap.ts

```diff
@@ -39,17 +41,30 @@ export default abstract class GlobeOrMap {
   private _tilesLoadingCountMax: number = 0;
   protected supportsPolylinesOnTerrain?: boolean;
 
+  // Fired when zoomTo is called
+  zoomToEvent: CesiumEvent = new CesiumEvent();
+
   // This is updated by Leaflet and Cesium objects.
   // Avoid duplicate mousemove events.  Why would we get duplicate mousemove events?  I'm glad you asked:
   // http://stackoverflow.com/questions/17818493/mousemove-event-repeating-every-second/17819113
   // I (Kevin Ring) see this consistently on my laptop when Windows Media Player is running.
   @observable mouseCoords: MouseCoords = new MouseCoords();
 
   abstract destroy(): void;
-  abstract zoomTo(
+
+  abstract doZoomTo(
     viewOrExtent: CameraView | Rectangle | Mappable,
     flightDurationSeconds: number
   ): void;
+
+  zoomTo(
+    viewOrExtent: CameraView | Rectangle | Mappable,
+    flightDurationSeconds: number
+  ): void {
+    this.zoomToEvent.raiseEvent();
+    this.doZoomTo(viewOrExtent, flightDurationSeconds);
+  }
+
```

Comment:
Raises an event when map zooms to a different location. Pedestrian mode will now disable itself when this event is fired.

Reply:
Where's the zoomToEvent handled by Pedestrian Mode? I can't find it 😬

Reply:
Yeah it is easy to miss:

https://github.com/TerriaJS/terriajs/blob/f59b7b7b72ca4217faf9ad6725c900e9ed6d517b/lib/ReactViews/Tools/PedestrianMode/PedestrianMode.tsx#L30-L32